### DISCUSSION TITLE
could we bump the go-jsonnet version to 0.17.0

### DISCUSSION THREAD

**Discussion by davidovich** [grafana/tanka#444] [Ideas]
I would have liked to propose this before 0.13.0 came out ! 

go-jsonnet [0.17.0 ](https://github.com/google/jsonnet/releases/tag/v0.17.0) offers significant manifestation speedups (sometimes 4x in non-scientific observations).

Did you consider upgrading this vital dependency ? :-)

**Comment by nabadger** [comment]
Would be useful. I started using `std.objectValuesAll` and `std.objectValues` in some of my jsonnet libs - which of course broke later when I ran against Tanka :grin:

**Reply by davidovich** [reply]
@Duologic took care of that in #445 🙂

**Reply by tombrk** [reply]
Thanks for the heads-up! Given this is a technical topic with a clear TODO, it actually would have been a good fit for an issue, instead of a dicussion.

Discussions are more of a support channel for questions, specific problems or vague ideas that need development before proposing them as issues.

Issues are for clear technical topics. This covers obviously bugs, but also feature requests.

Generally spoken, choose issues if you have clear suggestions that require code changed, use discussions if you don't understand something, or need other support.

**Comment by malcolmholmes** [comment]
FWIW I think this has been done in the 0.14.0 release.
